### DISCUSSION TITLE
Beta release

### DISCUSSION THREAD

**Discussion by butschster** [buggregator/app#71] [General]
Hello guys! 
Is there anyone, who is using buggregator 2.0-beta release?

**Comment by anugupta** [comment]
Hey, I just found out about this project, and yes I am. I've only done some simple testing, but certainly  dumping and using ray() methods is working fine in a Laravel project. 


The only thing I've noticed so far is the delete log button (the x) and clear screen buttons don't seem to do anything.  

Apart from that, this really does look amazing – great job!

**Reply by butschster** [reply]
Thank you, @anugupta!
